application/src/main/kotlin/org/gxf/standalonenotifyinggateway/coaphttpproxy/coap/configuration/properties/CoapProperties.kt

```diff
@@ -16,4 +16,5 @@ class CoapProperties(
     val maxPeerInactivityPeriod: Duration,
     val maxResourceBodySize: Int,
     val preferredBlockSize: Int,
+    val cipherSuites: List<String>
```

Comment:
I believe this should also work:
```suggestion
    val cipherSuites: List<CipherSuite>
```